enhancements/insights/conditional-data-gathering.md

```diff
@@ -187,7 +187,44 @@ the service.
 
 ## Design Details
 
-Empty
+On the initial start the conditional gatherer will try to fetch the config from the service
+by doing http(s) request to the appropriate endpoint `https://cloud.redhat.com/.../gathering_rules`.
+There is a timeout of one minute for the request. The response is then cached and sent again only when 12 hours pass
```

Comment:
Does it mean the Insights operator will be querying the service every 12h? Would it make sense to make this value configurable?

Reply:
Yes, why not